src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -655,6 +654,20 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        [RavenAction("/admin/backup-task/delay", "POST", AuthorizationStatus.Operator)]
```

Comment:
Shouldn't this be '/databases/*/admin/backup/delay' in OngoingTasksHandler?

Reply:
All endpoints with `/databases/*/admin` paths should have `ValidUser` or `DatabaseAdmin` authorization status, [but it should be](https://github.com/ravendb/ravendb/pull/15302#discussion_r1046787064) `Operator` for server-wide backups.